tests/config.c

```diff
@@ -226,37 +227,47 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
          config_set_gib("reclaim-threshold", cfg, reclaim_threshold) {}
          config_has_option("stats")
          {
-            for (cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+            for (int cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
                cfg[cfg_idx].use_stats = TRUE;
             }
          }
          config_has_option("no-stats")
          {
-            for (cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+            for (int cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
                cfg[cfg_idx].use_stats = FALSE;
             }
          }
          config_has_option("log")
          {
-            for (cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+            for (int cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
                cfg[cfg_idx].use_log = TRUE;
             }
          }
          config_has_option("no-log")
          {
-            for (cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+            for (int cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
                cfg[cfg_idx].use_log = FALSE;
             }
          }
          config_has_option("verbose-logging")
          {
-            cfg[cfg_idx].verbose_logging_enabled = TRUE;
```

Comment:
Here is the bug in the old code. Loop variable `cfg_idx` is uninitialized.

I had copy-cloned this pattern  a few lines below, for the new `--verbose-progress` parsing.

Fortunately, unit-test discovered this bug ... as it would fail in release build but succeed in debug build.